### ISSUE TITLE
Stucked on the FaceID screen on ioS

### ISSUE THREAD

**Issue by GBen1** [particl/copay#40]
Hello guys!

It seems there is an issue with the copay app since the last update on ioS:

![copay](https://user-images.githubusercontent.com/31903813/83687870-b7a30300-a5ec-11ea-8bbb-184c56583c36.jpg)

(nothing happen when I click on scan again)

**Comment by HenkSwardt** [comment]
Good day @GBen1 

Thank you for reporting this. I do see in the general channel that a user has confirmed this to be working for them. 

<jsolman@telegram>: Works fine for me
(re: :
Particl copay cannot be unlocked anymore on iOS I m stucked on the fa...)

Will it be possible for you to try and reinstall just to be certain?

Thank you

**Comment by GBen1** [comment]
Good morning Henk !

Thanks for your quick reply,
I reinstalled the app and I have been able to open it but I still think there is a problem.
After having access again to the app I tried to reactivate FaceID but this was not anymore accessible in the settings and it was greyed in the app.
I think I was stucked on this screen because I enabled faceID for this app before the last update (and that it s probably no longer available) as I don t have any issues with faceid on the others apps.
From now it shouldn t be a problem for me as I am just going to use a pin instead.

Have a great day 🙂


![6C45A6F2-C5DE-4103-9625-AE6C0177FFC2](https://user-images.githubusercontent.com/31903813/83738844-1b611680-a655-11ea-9d0f-054e71274497.png)
![40028540-1BE4-471A-AB56-C94710208EA3](https://user-images.githubusercontent.com/31903813/83738852-1e5c0700-a655-11ea-9814-7d4e9e1f803b.png)
